src/storage/SimpleResourceStore.ts

```diff
@@ -27,21 +21,29 @@ export class SimpleResourceStore implements ResourceStore {
    */
   public constructor(base: string) {
     this.base = base;
+
+    this.store = {
+      '': {
```

Comment:
This will need a comment; why empty string? What does this mean?
If special, perhaps you want to attach this key to a `Symbol` instead.